src/main/js/component/common/RefreshTableCellFormatter.js

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const RefreshTableCellFormatter = ({ handleButtonClicked, currentRowSelected }) => (
+const RefreshTableCellFormatter = ({handleButtonClicked, currentRowSelected}) => (
     <button
         className="btn btn-link editJobButton"
         type="button"
         title="Refresh"
-        onClick={() => {
+        onClick={(e) => {
+            e.stopPropagation();
```

Comment:
This is needed to stop the modal from popping up when you try to resend the notification